sspl_test/functional_tests/plans/alerts.pln

```diff
@@ -0,0 +1,23 @@
+alerts.node.test_node_disk_actuator

```

Comment:
all these test plans should exactly match with existing plans in v1 test framework

Reply:
Sure. Right now only sanity plan is exactly covered. I will get help from Palak to convert remaining test cases.